rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import resolve from 'rollup-plugin-node-resolve';
+import multiEntry from 'rollup-plugin-multi-entry';
+import ignore from 'rollup-plugin-ignore';
+
+import pkg from './package.json';
+import path from 'path';
+
+export default {
+  input: ['runtime/ts-build/runtime.js', 'runtime/handle.js'],
```

Comment:
Wondering what determines what we need to roll up, why we need handle.js, am inferring this was reverse engineered from failure in order to get the simple hello world test case below operational. Adding a TODO and talking with @raulverag further warranted here re: short term M1 and post-M1 plans?

Reply:
The rollup-plugin-multi-entry is strictly not necessary at this point other than to show how we can support this capability.  I thought it would be good to have that ready for future use.  I don't consider it YAGNI.

Reply:
Yeah, this is fine for now; we are definitely going to need it to support M1.

Post-M1, Raul's work will build out the runtime object to the point where we can remove all the extra entry points; this list can in fact be a driver for what he's doing.